internal/command/postgres/postgres.go

```diff
@@ -18,10 +18,10 @@ import (
 
 func New() *cobra.Command {
 	const (
-		short      = `Deprecated: Unmanaged Postgres cluster commands'`
-		deprecated = "DEPRECATED: Unmanaged Fly Postgres is deprecated in favor of 'fly mpg' (Managed Postgres).\n" +
+		short  = `Deprecated: Unmanaged Postgres cluster commands'`
```

Comment:
Should we also change the `Deprecated:` text here to say `Warning:` or maybe `Notice:` instead?